universal-application-tool-0.0.1/conf/application.conf

```diff
@@ -77,6 +77,8 @@ adfs.discovery_uri = "https://sts.seattle.gov/adfs/.well-known/openid-configurat
 adfs.discovery_uri = ${?ADFS_DISCOVERY_URI}
 adfs.admin_group = "ad\\ITD_CiviForm_Admins_Test"
 adfs.admin_group = ${?ADFS_GLOBAL_ADMIN_GROUP}
+adfs.additional_scopes = "allatclaims"
```

Comment:
suggestion (blocking): please add a comment specifying how tokens in the scopes list here should be separated

Reply:
added a lil comment